app/cmd/plugin_search.go

```diff
@@ -93,3 +144,30 @@ func (o *PluginSearchOption) Output(obj interface{}) (data []byte, err error) {
 	}
 	return
 }
+
+func formatTab(table *util.Table, i int, plugin client.CenterPlugin) {
+	installed := plugin.Installed
+	if installed != (client.InstalledPlugin{}) {
+		if len(plugin.Version) > 6 && len(installed.Version) > 6 {
```

Comment:
What about that if the user wants to see the full version string?

Reply:
Some version string maybe has so long. 
Show that it may affect the effect. So I trim it

Reply:
Yes. But my question still exists.

Reply:
I make it show full strings already